bqplot/axes.py

```diff
@@ -78,7 +78,7 @@ class Axis(BaseAxis):
     label: string (default: '')
         The axis label
     tick_format: string or None (default: '')
-        The tick format for the axis.
+        The tick format for the axis, for dates use d3 string formatting.
```

Comment:
I'm suggesting that this be made explicit here, as well as more generally. If I'm trying to figure out how to use a optional parameter on a function, I'm likely to look at its docstring. It wouldn't have helped to have a more general statement if I wouldn't have known to look for such a more general statement or where to look for it.

It should definitely be a general statement in a tutorial or example somewhere. But, I think of docstrings as more reference documentation than expository/educational documentation. Accordingly, either having information as close as possible to where the information is needed (even if that results in redundant information) or an explicit reference to where the information can be found is usually a good policy.